hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/table/action/rollback/TestMergeOnReadRollbackActionExecutor.java

```diff
@@ -126,7 +125,11 @@ public void testMergeOnReadRollbackActionExecutor(boolean isUsingMarkers) throws
     for (Map.Entry<String, HoodieRollbackPartitionMetadata> entry : rollbackMetadata.entrySet()) {
       HoodieRollbackPartitionMetadata meta = entry.getValue();
       assertTrue(meta.getFailedDeleteFiles() == null || meta.getFailedDeleteFiles().size() == 0);
-      assertTrue(meta.getSuccessDeleteFiles() == null || meta.getSuccessDeleteFiles().size() == 0);
+      if (isUsingMarkers) {
+        assertTrue(meta.getSuccessDeleteFiles() == null || meta.getSuccessDeleteFiles().size() == 0);
+      } else {
+        assertFalse(meta.getSuccessDeleteFiles() == null || meta.getSuccessDeleteFiles().size() == 0);
+      }
```

Comment:
this is existing logic: are we able to fix the assertion conditions by not using `||` here and just check `meta.getSuccessDeleteFiles().size()`?  i'm curious to know why we need to null-check `meta.getSuccessDeleteFiles()` and `meta.getFailedDeleteFiles()`.